server/storage/mvcc/index.go

```diff
@@ -107,6 +105,9 @@ func (ti *treeIndex) unsafeVisit(key, end []byte, f func(ki *keyIndex) bool) {
 	})
 }
 
+// Revisions returns limited number of revisions from key(including) to end(excluding)
+// at the given rev. The returned slice is sorted in the order of key.
+// There is no limit if limit <= 0.
```

Comment:
It seems also important that 'total' is not capped by the limit and reflects all the revisions.

Reply:
Comments:
1. `including` to `included`, and `excluding` to `excluded`?  This applies to the comment for method `CountRevisions` as well.
2. adding comment something like "The second return parameter `total` isn't capped by the `limit`, and reflects all the expected revisions".

I'd defer to @spzala to correct the syntax of the comment.

Reply:
Thanks for review, update as you commented.